.github/workflows/ci.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
       typescript: ${{ steps.filter.outputs.typescript }}
     steps:
       - uses: actions/checkout@v3
-      - uses: dorny/paths-filter@v2
+      - uses: dorny/paths-filter@4067d885736b84de7c414f582ac45897079b0a78
```

Comment:
This is one PR after the last tagged version, but it's a doc-only change, so it shouldn't matter.